numpy/linalg/lapack_lite/README.rst

```diff
@@ -12,15 +12,17 @@ automatically from a directory of LAPACK source files.
 You'll need `plex 2.0.0dev`_, available from PyPI, installed to do the
 appropriate scrubbing. As of writing, **this is only available for python 2.7**,
 and is unlikely to ever be ported to python 3.
+As a result, all the python scripts in this directory must remain compatible with
+Python 2.7, even though NumPy itself no longer supports this version.
```

Comment:
Feel free to tweak the wording to whatever you think drives the point home, I wasn't able to make this as a suggestion on github.

Reply:
I would add "until these scripts are rewritten to use something other than ``plex``." We do want to move towards that.

Reply:
Added too.